From 84b024456a94f294017cd654d8165a834942e6c5 Mon Sep 17 00:00:00 2001
From: Cesare Mercurio <cesare.mercurio@gmail.com>
Date: Sun, 17 Dec 2023 07:44:44 -0800
Subject: [PATCH 2/2] VVI-node-addon-api : Add build file for bzlmod

Summary:
Adds BUILD file for bzlmod registry
Test Plan:
integration with vvplatform
---
 BUILD | 10 ++++++++++
 1 file changed, 10 insertions(+)
 create mode 100644 BUILD

diff --git a/BUILD b/BUILD
new file mode 100644
index 0000000..8f387d4
--- /dev/null
+++ b/BUILD
@@ -0,0 +1,10 @@
+load("@rules_cc//cc:defs.bzl", "cc_library")
+
+cc_library(
+    name = "node-addon-api",
+    hdrs = glob(["*.h*",]),
+    includes = [
+        ".",
+    ],
+    visibility = ["//visibility:public"],
+)
\ No newline at end of file
-- 
2.34.1

